R/utilities-data-repository.R

```diff
@@ -8,8 +8,7 @@
 #'
 #' obsData <- loadDataRepositoryFromPKML(filePath)
 #' metaData <- obsData$metaData
-#' @export
-loadDataRepositoryFromPKML <- function(filePath) {
+.loadDataRepositoryFromPKML <- function(filePath) {

```

Comment:
I propose not to export this function. The class DataRepository is also not exported, so user interaction with it should be as minimal as possible.

Reply:
Why is the DataRepository class not exported ? Aren't they all exported by default?

Reply:
Nope, you have to add @export to class definition.